### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Misha-Ov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Project Structure
+- **Home Page**: Overview of the destination with popular places, customer reviews, and a call to action.
+- **Country Page**: Detailed information about the country, recommended itineraries, and an interactive map.
+- **Contact Page**: Contact form, office location, and a FAQ section.
+
+## How to Run Locally
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/yourusername/HW14_travel_site.git
```
Comment (fullstack-and-test) [inline README.md]: cool start 👍

please replace with https://github.com/Misha-Ov/SEAsia.git

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+3. **Tabs**: JavaScript-driven tabs for displaying organized content.
+4. **Fixed Navigation**: The header menu remains visible while scrolling.
+5. **Interactive Map**: Google Maps integration to display key locations.
+6. **CSS Animations**: Hover effects and transitions on buttons and cards.
+7. **Responsive Design**: Site adapts to different screen sizes using percentages for widths and Bootstrap's grid system.
+8. **Contact Form**: Includes form validation with smooth submission feedback.
+
+## Project Structure
+- **Home Page**: Overview of the destination with popular places, customer reviews, and a call to action.
+- **Country Page**: Detailed information about the country, recommended itineraries, and an interactive map.
```
Comment (fullstack-and-test) [inline README.md]: + Tour description page

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# HW14 Travel Agency Website
```
Comment (fullstack-and-test) [inline README.md]: please replace with SEAsia  🎉

**Review by fullstack-and-test — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Misha-Ov) [reply README.md]: ✅ Done

File: README.md (hunk 2 above)
Comment (Misha-Ov) [reply README.md]: ✅ Done

File: README.md (hunk 3 above)
Comment (Misha-Ov) [reply README.md]: ✅ Done

**Review by fullstack-and-test — COMMENTED** [review]
ok

**Pull request merged**